inspirehep/modules/search/query.py

```diff
@@ -71,16 +93,9 @@ def build(self, **kwargs):
                     "multi_match": {
                         "query": self._query,
                         "zero_terms_query": "all",
-                        "fields": [
-                            "title^3",
-                            "title.raw^10",
-                            "abstract^2",
-                            "abstract.raw^4",
-                            "author^10",
-                            "author.raw^15",
-                            "reportnumber^10",
-                            "eprint^10",
-                            "doi^10"]}}
+                        "fields": google_search_fields[self.index]
```

Comment:
Design: this could even be a method call instead of a lookup, so we can inject logic if/when we A/B test search.